src/daemon/dlt-daemon.c

```diff
@@ -259,6 +271,14 @@ int option_handling(DltDaemonLocal *daemon_local, int argc, char *argv[])
         }
 #endif
 
+#ifdef DLT_DAEMON_USE_UNIX_SOCKET_IPC
+        case 'u':
+        {
+            strncpy(daemon_local->flags.appSockPath, optarg, NAME_MAX);
```

Comment:
Could you please use DLT_IPC_PATH_MAX here instead of NAME_MAX.

Reply:
ok